### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by rgarben**
add landing message. #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
     <title> Welcome to Potionfy</title>
 </head>
 <body>
-
+	<h1>Welcome to Potionfy! enjoy your stay.</h1>
```
Comment (cristina041006) [inline index.html]: Could you remove the "Enjoy your stay" string and make the "Welcome to Potionfy!" string red?

**Comment by rgarben** [comment]
sure, can do!

**Review by cristina041006 — CHANGES_REQUESTED** [review]
Please, changue the code as per my comments on the review

**Review by cristina041006 — APPROVED** [review]
Well done Harry!

**Comment by rgarben** [comment]
done

**Pull request merged**